examples/cp4d_4.0/README.md

```diff
@@ -0,0 +1,95 @@
+# Example to provision CP4Data Terraform module
+
+## Run using IBM Cloud Schematics
+
+For instructions to run these examples using IBM Schematics go [here](../Using_Schematics.md)
+
+For more information on IBM Schematics, refer [here](https://cloud.ibm.com/docs/schematics?topic=schematics-get-started-terraform).
+
+## Run using local Terraform Client
+
+For instructions to run using the local Terraform Client on your local machine go [here](../Using_Terraform.md)
+customizing these values in the `terraform.tfvars` file:
+
+```hcl
+resource_group_name = "Default"
+
+// ROKS cluster parameters:
+cluster_id          = "******************"
+region                = "us-south"
+on_vpc              = false
+
+// Prereqs
+worker_node_flavor = "bc3.16x64"
+
+// Entitled Registry parameters:
+entitled_registry_key        = "******************"
+entitled_registry_user_email = "john.doe@email.com"
+
+// CP4D License Acceptance
+accept_cpd_license = true
+
+// Parameters to install submodules
+install_watson_knowledge_catalog = false
+install_watson_studio            = false
+install_watson_machine_learning  = false
+install_watson_open_scale        = false
+install_data_virtualization      = false
+install_streams                  = false
+install_analytics_dashboard      = false
+install_spark                    = false
+install_db2_warehouse            = false
+install_db2_data_gate            = false
+install_big_sql                  = false
+install_rstudio                  = false
+install_db2_data_management      = false
```

Comment:
Either these should be like the names in `variables.tf` (so become `install_wkc` for example.) Or the stuff in `main.tf` (for example at `./main.tf:75` should be updated so it reads from these new and more descriptive names.

Reply:
```suggestion
  install_wsl         = false
  install_aiopenscale = false
  install_wml         = false
  install_wkc         = false
  install_dv          = false
  install_spss        = false
  install_cde         = false
  install_spark       = false
  install_dods        = false
  install_ca          = false
  install_ds          = false
  install_db2oltp     = false
  install_db2wh       = false
  install_big_sql     = false
  install_wsruntime   = false
```